### DISCUSSION TITLE
[5.x]: Add a "Delete" action to entry type edit pages

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/cms#14983] [Ideas] [answered]
On field edit pages, there is a "Delete" action available in the "Save" button disclosure menu:

![CleanShot 2024-05-13 at 11 19 21](https://github.com/craftcms/cms/assets/298510/98b360fb-5ae2-4d3a-9f06-c0df2dfa3122)

However, for entry types there is no "Delete" (or "Save and add another"):

![CleanShot 2024-05-13 at 11 21 04](https://github.com/craftcms/cms/assets/298510/0bb7ce22-5cd3-4064-bd4a-59c4f93cec88)

_In attempting to clean up (i.e. delete) unused entry types, edit pages are a good place to gauge whether or not a specific entry type is in use anywhere. To that end, it'd be great if it was possible to delete the entry type from this view, instead of having to go back to the entry type list page, find the correct, unused type in the table and delete it there._

**Comment by brandonkelly** [comment, marked as answer]
Added for 5.2! (288f18613c5111c77bd6fcd37207183e6282201a)

**Reply by brandonkelly** [reply]
5.2.0-beta.1 is out with that change.
